main.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// A Recipe Struct allows you to insert recipe documents into your
+// collection
+
+type Recipe struct {
+	Name              string
+	Ingredients       []string
+	PrepTimeInMinutes int `json:"prepTimeInMinutes" bson:"prepTimeInMinutes"`
+}
+
+func main() {
+
+	// TODO:
+	// Replace the placeholder connection string below with your
+	// Altas cluster specifics. Be sure it includes
+	// a valid username and password! Note that in a production environment,
+	// you do not want to store your password in plain-text here.
+	var mongoUri = "<Your Atlas Connection String>"
+
+	// CONNECT TO YOUR ATLAS CLUSTER:
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(
+		mongoUri,
+	))
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = client.Ping(ctx, nil)
+
+	if err != nil {
+		log.Fatal("There was a problem connecting to your Atlas cluster. Check that the URI includes a validusername and password, and that your IP address has been whitelisted. Error: ", err)
```

Comment:
validusername => valid username

Reply:
whitelisted => added to the access list

Reply:
This should also probably be `panic` since the rest of the application will not run properly.

Reply:
👍